### DISCUSSION TITLE
If you love python leave a 🐍 here

### DISCUSSION THREAD

**Discussion by gravitytrope** [community/events#33] [Community]
Hello from the Watch Party / Community Hangout at Women Who Code Python 🐍 👋

**Comment by jricciardi** [comment]
🐍🐍🐍

**Comment by K0I0S-Codes** [comment]
🐍

**Comment by entcheva** [comment]
🐍🐍🐍

**Comment by ossanna16** [comment]
Hello from PyLadies London 🐍

**Comment by queenofcorgis** [comment]
🐍 !

**Comment by SirDre** [comment]
🐍👀
![image](https://user-images.githubusercontent.com/1511532/200919787-0150b650-3df1-4d1d-8cfc-a6e7629713b2.png)

**Comment by GalletaOreo98** [comment]
🐍❤

**Comment by lauraann87** [comment]
🐍🐍🐍

**Comment by JackEAllen** [comment]
🐍
     🐍 🐍
    🐍 🐍 🐍

**Comment by ItsNyoty** [comment]
🐍

**Comment by Justintime50** [comment]
🐍 yesssssssthsthsthsthsths 😉

**Comment by KazimMir** [comment]
🐍🐍🐍🐍🐍            🐍                 🐍         🐍🐍🐍🐍🐍     🐍              🐍                    🐍🐍🐍        🐍                  🐍                         
🐍                   🐍            🐍              🐍                   🐍               🐍              🐍              🐍              🐍    🐍 🐍             🐍 
🐍                  🐍                  🐍         🐍                   🐍               🐍              🐍               🐍              🐍   🐍     🐍         🐍              
🐍🐍🐍 🐍 🐍                       🐍🐍🐍                   🐍              🐍🐍🐍🐍🐍              🐍              🐍    🐍        🐍      🐍 
🐍                                                        🐍                   🐍              🐍              🐍              🐍              🐍    🐍          🐍    🐍 
🐍                                                        🐍                   🐍              🐍              🐍              🐍              🐍    🐍            🐍  🐍 
🐍                                                        🐍                   🐍              🐍              🐍              🐍              🐍    🐍             🐍 🐍 
🐍                                                        🐍                   🐍              🐍              🐍                  🐍🐍🐍         🐍                   🐍   🐍  🐍  🐍

**Reply by Patecatl848** [reply]
Nice

**Comment by KazimMir** [comment]
![image](https://user-images.githubusercontent.com/108958999/200924630-3862c93f-db3e-4ef6-b625-e91f3d332bcf.png)

Oh no...
![image](https://user-images.githubusercontent.com/108958999/200924739-134fd128-5ffe-4900-b1ea-1844e205c1e7.png)

**Reply by Patecatl848** [reply]
Very nice

**Comment by Xp101T7** [comment]
🐍🐍🐍🐍

**Comment by MegaChestercat** [comment]
🐍

**Comment by JonesKYM** [comment]
🐍🐍🐍

**Comment by Buedenbender** [comment]
🐍 ❤️

**Comment by RebeccaBecky** [comment]
*/   ,%%%%%%%%%%#%%%##(/,                                      
                 .//  .%%%%%%%%/  .(%%%%%%%%(,                                  
               ,/((((((#%%%%%%%#  ,#%%%%%%%%%%#,                                
                        .(%%%%%%%%%%%%%%%%%%%%%%(.                              
                            ............../#%%%%%#                              
                                           *(#%%%%/                             
                                           .(#%%%%%                             
                                            /#%%%%%.                            
                                            /#%%%%%.                            
                                           .(#%%%%%.                            
                      .*(#%%%%%%%%%%%%%%%%%%%%%%%%%*                            
                  .(%%%%%%%%%###############%%%%%%#((##/.                       
                 *#%%%%%%#(((((((((((((((((#%%%%%#(((%%%%#*                     
                 *(#%%%%%#/*,,,,,,,,,,,*(%%%%%%#(//(#%%%%%%/                    
                  /(((##%%%%%%%%%%%%%%%%%%%#((((,    #%%%%%#                    
                    ,/((((((((((((((((((((((/,      /%%%%%#/                    
                          ...,,,,,,,,,,..      .*/#%%%%%#((.                    
                .*(#%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%###(((*                      
              ,#%%%%%%%%%%%%%%%%%%%%%%%######(((((((((*                         
             .(((((((((((((((((((((((((((((((((*,.

**Comment by hBk-m3** [comment]
🐍🐍🐍🐍🐍Loooovvvee

**Comment by mishmanners** [comment]
🐍🐍🐍

**Comment by ming-tsai** [comment]
🐍

**Comment by Patecatl848** [comment]
SSSSSSSS...🐍🐍🐍🐍🐍
![image](https://user-images.githubusercontent.com/61675959/200968867-54778e72-2eea-4b2b-8394-669e531bbba9.png)

**Comment by airtower-luna** [comment]
Best snake! :snake:

**Comment by Puri12** [comment]
:snake:

**Comment by KevinT02** [comment]
🐍

**Comment by kishorekumar-it19** [comment]
![image](https://user-images.githubusercontent.com/53468417/201172545-aa913a2f-278d-41e9-bfd9-6bc283c99e47.png)

**Comment by k-lakshmikanth** [comment]
🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍🐍

**Comment by JWCook** [comment]
![🐍🐍🐍🐍🐍🐍](https://user-images.githubusercontent.com/419936/201174538-b8cc8c56-356e-4d03-a2be-0d014965b9c7.jpg)

**Comment by ghostinhershell** [comment]
🐍🐍🐍

Can anyone write 'PYTHON' only using the 🐍 emoji?

**Reply by JWCook** [reply]
A goofy roundabout way to do it:
```python
def join_char_offsets(input_char, offsets):
    return ''.join([chr(ord(input_char) - i) for i in offsets])

join_char_offsets('🐍', [127933, 127924, 127929, 127941, 127934, 127935])
```

**Comment by NightOwlDevs** [comment]
![image](https://user-images.githubusercontent.com/87749061/201185049-6b9b643f-4300-4deb-a9a3-d62bccdcedcd.png)

**Comment by dhrubasaha08** [comment]
🐍

**Comment by hrt127** [comment]
>> And we danced....and we cried
Having a merry .... Magical puppets singing... kinda </summary>

> | Science		        |  Fictionoooor       |
> |------------------------ |------------------- |
> |Snape	  Snape  	 	| Severus  ❓          |
> |Snape       🐍              | Severus 🐍          |
> |  (🎺 ,,,,🧙🏾🏾)             |    Doooorrrrr        |
  
</details>

**Comment by abbycabs** [comment]
I loved @blackgirlbytes "Pythonista" shoutout during the Day 1 Keynote ❤️

**Comment by jagadeeshi2i** [comment]
🐍

**Comment by S-PADMANABAN-2001** [comment]
🐍

**Comment by NicoleAshley88** [comment]
> 🐍🐍🐍

**Reply by khaleefa222** [reply]
k

**Comment by AKRIKEZMOHAMED** [comment]
Goodevning...i am a student in ALX SE program. But i feel bad.i dont finding the direction . If having the advices. Thanks.

**Reply by ossanna16** [reply]
Hi there @AKRIKEZMOHAMED and welcome to our community! What sort of direction are you looking for? Learning resources? Projects to collaborate on? If you are a bit more specific, perhaps I can make some suggestions 🙂

**Comment by AKRIKEZMOHAMED** [comment]
hi Ms Anas; thank you very much friend. the direction from the best way of alx program. i fell bad and i can't go just to 10 por cent . but the conditions the near of 80 por cent.  and thanks for you again.
________________________________
De : Anna Ossowski ***@***.***>
Envoyé : lundi 20 mars 2023 12:51
À : community/events ***@***.***>
Cc : AKRIKEZMOHAMED ***@***.***>; Mention ***@***.***>
Objet : Re: [community/events] If you love python leave a 🐍 here (Discussion #33)


Hi there @AKRIKEZMOHAMED<https://github.com/AKRIKEZMOHAMED> and welcome to our community! What sort of direction are you looking for? Learning resources? Projects to collaborate on? If you are a bit more specific, perhaps I can make some suggestions 🙂

**Reply by ossanna16** [reply]
@AKRIKEZMOHAMED Can you please explain what the ALX Program is? I've never heard of it.

**Comment by AKRIKEZMOHAMED** [comment]
The program of engineering of programers peoples and talens mens in the technologie and languages programmation.

Télécharger Outlook pour Android<https://aka.ms/ghei36>
________________________________
